keywords/LiteServiOS.py

```diff
@@ -32,8 +61,8 @@ def install(self):
         if self.storage_engine != "SQLite":
             raise LiteServError("https://github.com/couchbaselabs/liteserv-ios/issues/1")
 
-        package_name = "couchbase-lite-ios-liteserv-{}.app".format(self.version_build)
-        app_path = "{}/{}".format(BINARY_DIR, package_name)
+        package_name = "LiteServ-iOS.app"
+        app_path = "{}/{}/{}".format(BINARY_DIR, "LiteServ-iOS", package_name)
```

Comment:
This will be fine for now. I think we will have to fix a few things when iOS is building in CI